tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -130,6 +130,126 @@ async def test_reborn_v2_composer_accepts_draft_while_run_is_processing(reborn_v
+            if (!stream) throw new Error("no EventSource stream is open");
+            stream.readyState = 2;
+            if (typeof stream.onerror !== "function") {
+              throw new Error("EventSource has no error handler");
+            }
+            stream.onerror(new Event("error"));
+          };
+        })();
+        """
+    )
+
+    async def fulfill_json(route, body, status=200):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `fulfill_json`

Add return type annotation: `None`

(ANN202)